README.md

```diff
@@ -65,36 +63,52 @@ $ docker run -ti --rm -v $PWD:/data -w /data ghcr.io/lcsb-biocore/docker/fbcmode
 
 ## MEMOTE-style tests
 
-You can use a number of tests that automatically check various basic quality
-characteristics of a constraint-based metabolic model; the suite available in
-FBCModelTests.jl is inspired by [MEMOTE](https://memote.readthedocs.io/).
+The primary entry point for the [MEMOTE](https://memote.readthedocs.io/) test
+suit implemented here is the function `run_tests`. When building a model, it is
+most convenient to incorporate it into the CI of the model. Another option is to
+use the command line functionality, and save the output for later analysis.
 
 To run the test suite on a toy model, use `run_tests`:
 ```julia
-using FBCModelTests, Tulip
-FBCModelTests.Memote.run_tests("e_coli_core.json", Tulip.Optimizer)
+using FBCModelTests, Gurobi
+FBCModelTests.Memote.run_tests("e_coli_core.json", Gurobi.Optimizer)
 ```
-
-An overview of the model properties can be also gathered in a `Dict` for
-mechanical inspection (or saving into JSON):
+Any optimizer supported by [JuMP](https://jump.dev/) can be used. The output of
+`run_tests` is the standard Julia unit testing scheme. However, in the repl the
+full output is usually truncated, and only a summary is shown. If you want more
+details about where your model failed certain tests, it is best to capture the
+output, and save it to a file. A convenient way to do this is with
+[ansi2html](https://github.com/agnoster/ansi2html).
+
+An example workflow entails creating a testing script, called `test.jl` as an
+example, with the tests you want to run:
 ```julia
-using FBCModelTests, Tulip
-structured_report = FBCModelTests.Memote.generate_report("e_coli_core.json, Tulip.Optimizer)
-
-using JSON
-open("my_e_coli_core_report.json", "w") do io
-    JSON.print(io, report)
-end
+using FBCModelTests, Gurobi
+FBCModelTests.Memote.run_tests("e_coli_core.json", Gurobi.Optimizer)
+```
+Then, from the command line (not within Julia) run:
+```
+julia --color=yes 'test.jl' > test.out
```

Comment:
Could you actually put the script into the `bin/` directory (like the other scripts) ?

That way people can do just
```sh
bin/run-memote --color=yes e_coli_core.json
# or maybe
bin/make-memote-json-report e_coli_core.json
```
or so